lib/improver/cli/__init__.py

```diff
@@ -220,6 +220,36 @@ def with_intermediate_output(wrapped, *args, intermediate_output=None,
     return result
 
 
+def create_constrained_inputcube_converter(*constraints):
```

Comment:
Returns a `CubeList`, so can we rename this (and the inner function) as `inputcubelist` please?

Reply:
Done